pkgs/by-name/ap/apparmor-parser/package.nix

```
@@ -28,7 +28,7 @@ stdenv.mkDerivation (finalAttrs: {
     substituteInPlace rc.apparmor.functions \
       --replace-fail "/sbin/apparmor_parser" "$out/bin/apparmor_parser" # FIXME
     substituteInPlace rc.apparmor.functions \
-      --replace-fail "/usr/sbin/aa-status" '$(which aa-status)'
+      --replace-fail "/usr/sbin/aa-status" '$(command -v aa-status)'
```

Why not remove reliance on $PATH altogether? This would still assume `apparmor-bin-utils` to be in path, which might be an assumption that still breaks. Maybe this is a better approach:
```suggestion
      --replace-fail "/usr/sbin/aa-status" "${lib.getExe' apparmor-bin-utils "aa-status"}"
```